### PR TITLE
Markl update private sms tutorial

### PR DESCRIPTION
**Pull request by marklewin**
Make use of ES6 features and use Node promises instead of external dependency on `bluebird`

### CODE REVIEW

**Comment by mheap** [comment]
LGTM from a cursory read through, but I've not had chance to clone the repo and test it out. Could another reviewer test the app?

**Comment by marklewin** [comment]
Simplified tutorial code by removing everything that is not fundamental to the use case - such as Number Insight checking and automatic provisioning of virtual numbers.

File: example.env [hunk 1]
```diff
@@ -1,5 +1,4 @@
 NEXMO_API_KEY=
 NEXMO_API_SECRET=
-NEXMO_DEBUG=true
-PROVISIONED_NUMBERS=[{"country":"2_CHAR_CODE","msisdn":"NUMBER"}]
-SMS_WEBHOOK_URL=
+VIRTUAL_NUMBER_A=
+VIRTUAL_NUMBER_B=
```
Comment (mheap) [inline example.env]: Why do we need two virtual numbers here? We can link two numbers to a single number and then distribute the message based on the `from` field

**Review by mheap — CHANGES_REQUESTED** [review]

File: server.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const app = express();
+app.set('port', (process.env.PORT || 3000));
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.listen(app.get('port'), function () {
+    console.log('SMS Proxy App listening on port', app.get('port'));
+});
+
+const config = require('dotenv').config();
```
Comment (mheap) [inline server.js]: In most apps you don't assign `dotenv.config()` to anything. You just call `require('dotenv').config();` as the first line of your app and use `process.env.VARIABLE_NAME` where needed

File: example.env (hunk 1 above)
Comment (marklewin) [reply example.env]: Great idea! Refactored code to achieve this

File: server.js (hunk 2 above)
Comment (marklewin) [reply server.js]: Changed to use `process.env.VARIABLE_NAME`.

**Review by mheap — APPROVED** [review]

**Comment by lornajane** [comment]
@marklewin As discussed I've flipped the incoming webhook to be GET rather than POST, simply because it's an account-wide setting and is commonly GET. This is a great update and if you're happy, I'll merge

**Review by lornajane — APPROVED** [review]
Much simpler, thanks!

**Comment by marklewin** [comment]
Thanks for this. Looks great!

**Pull request merged**